code/modules/hydroponics/hydro_tools.dm

```diff
@@ -17,9 +17,9 @@
 	icon = 'icons/obj/device.dmi'
 	icon_state = "hydro"
 	item_state = "analyzer"
-
-/obj/item/device/analyzer/plant_analyzer/attack_self(mob/user as mob)
-	return 0
+	var/form_title //Descriptive title of the last plant scanned, example: mutant watermelon (#81)
+	var/last_data  //Stores the entire last scan, for printing purposes.
+	var/last_print = 0 //When was the last printing, works as a cooldown to prevent paperspam
```

Comment:
A `/tmp` variable is a variable that's "temporary", essentially it causes the map writer to not write this.